controllers/users.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../models/user');
+
+const userSerializer = require('../serializers/user');
+
+exports.create = async (req, res) => {
+  const user = await User.create(req.body);
+  if (user.errors) {
+    res.json({ user });
+  } else {
+    const serializedUser = await userSerializer(user);
+    req.session.userId = user.id;
+    res.json({ user: serializedUser });
+  }
+};
+
+exports.me = async (req, res, next) => {
+  if (!req.session.userId) {
```

Comment:
This seems kinda authy to me... should be moved to some current user middleware and called above the route to be more clear.

Reply:
Love this idea.